src/DataBoxEdge/DataBoxEdge/Common/Cmdlets/Bandwidths/DataBoxEdgeBandwidthScheduleGetCmdlet.cs

```diff
@@ -0,0 +1,156 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
this file and "src/DataBoxEdge/DataBoxEdge/Common/Cmdlets/Bandwidth/DataBoxEdgeBandwidthScheduleGetCmdletBase.cs"
doesn't look like the same, it's hard to tell what you changed.